packages/flutter_tools/lib/src/commands/listen.dart

```diff
@@ -45,20 +45,20 @@ class ListenCommand extends StartCommandBase {
       try {
         runCheckedSync(<String>['which', 'fswatch']);
       } catch (e) {
-        logging.severe('"listen" command is only useful if you have installed '
+        logging.severe('"listen" command requires '
             'fswatch on Mac.  Run "brew install fswatch" to install it with '
             'homebrew.');
-        return null;
+        exit(1);
```

Comment:
I think it's more idiomatic to continue returning null here and below, and just `return 1` from the call site when this returns null.